### PR TITLE
Add README

### LINKED ISSUE #60: Create a README file
The repo could use a README to explain a little bit about why we have a public blog, plus setup instructions for anyone working on it.

### PR DESCRIPTION
**Pull request by kapowaz**
* Added a basic README with setup instructions and general information about adding guidelines / blog posts
* Added some sample guidelines / blog post files as templates

Fixes #60.

### CODE REVIEW

**Comment by mezis** [comment]
:+1:

File: _guidelines/_example-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+We use the Rouge syntax highlighter, which supports numerous[^not-jsx] modern
+languages, and will give you nice output. Using this is as simple as:
+
+```markdown
+  ```language
+  # your code goes here
+  ```
+```
+
+[^not-jsx]: Sadly, JSX is not yet supported; you should simply omit the language identifier in this case.
```
Comment (lededje) [inline _guidelines/_example-guidelines.md]: Are we trying to adhere to an 80 character column limit?

Comment (kapowaz) [inline _guidelines/_example-guidelines.md]: Wherever possible, yeah — annoyingly Markdown footnote references break if you split them over multiple lines. 😞

**Comment by martyphee** [comment]
LGTM

**Pull request merged**